### ISSUE TITLE
Exception when inserting blank line at the end of a program

### ISSUE THREAD

**Issue by LANOYDO** [TypeCobolTeam/TypeCobol#1572]
**Describe the bug**
Adding a new blank line at the end of a program file throws an exception, when this program file has been saved with more than one finishing blank line.

**To Reproduce**
Save this program with more than one finishing blank line
````cob
       identification division.
       program-id. MyPGM.
       data division.
       working-storage section.
       01 Type1 typedef strict public.
          05 chaine pic X.
       01 var1 type type1.
      
       procedure division.
           call foo output var1.
           .
       declare foo  output b type type1.
       end-declare. 
       end program MyPGM.
````
Re-open this program (one blank line has been deleted).
Place the cursor at the end and add a new blank line.  Following error message should appear.

![image](https://user-images.githubusercontent.com/55186270/70240732-0e2ea200-176e-11ea-82a0-133484af2f6f.png)

**Expected behavior**
The case should be handled.

**Comment by smedilol** [comment]
See if #1591 resolve this issue

**Comment by rooksdo** [comment]
Fixed in v1.4.2.
